### DISCUSSION TITLE
How to import music from Apple music?

### DISCUSSION THREAD

**Discussion by RomanSelischev** [z-huang/music#466] [Q&A] [answered]
I want to transfer all the music to this application,  how do i move all the music from Apple music to this app (preferably **without** downloading it)?

usually it is done through online services where you need to log in from two accounts.

**Comment by z-huang** [comment, marked as answer]
Maybe you can first transfer your music to a YouTube playlist, and then import the playlist in InnerTune.

**Reply by RomanSelischev** [reply]
please tell me how to import YouTube playlist in InnerTune? I only found the restore button (Which asks for the path to the database). Can I somehow transfer music without downloading it?

**Reply by z-huang** [reply]
Someone uses this [https://www.tunemymusic.com/](https://www.tunemymusic.com/).

**Reply by RomanSelischev** [reply]
@z-huang this i did, how do i import playlist from youtube music to InnerTune?

**Reply by z-huang** [reply]
Make your playlist public, and you can search it in InnerTune.

**Reply by RomanSelischev** [reply]
Thx 👍
